packages/thicket-webapp/README.md

```diff
@@ -10,7 +10,13 @@ By setting the corresponding `env` var the IPFS instance will connect to the def
 - `WRTC_IPFS` - will use IPFS’s [`webrtc-star-signal`](https://wrtc-star.discovery.libp2p.io) server. Default value if no value is set.
 
 ```sh
+# to be sure have nodejs 8
+#goto root project
```

Comment:
Would you mind updating your PR to the following texts:

Requires node version __ (let's confirm the correct version)

```sh
# please make sure you have bootstrapped all the packages accordingly
# from repo root run (cd ../../)
yarn
yarn bootstrap

# then in packages/thicket-webapp (cd packages/thicket-webapp)
yarn start
```

Reply:
yep make sense. do you want update my change with that ?